### ISSUE TITLE
Clarify which config parameters can be passed to refresh()

### ISSUE THREAD

**Issue by pbowyer** [toorshia/justgage#241]
I was really excited to see config parameters can now be passed to `refresh()`, hoping this would let me change the colour of the text when the dial was completely full (it's hard to tell on a small gauge if you've got there or are a couple of items off being 'full').

However, all my attempts to change settings have failed. For example:
```
gauge.refresh(100, null, {titleFontColor: '#ffcc00'});
```
```
gauge.refresh(105, false, {labelFontColor: '#ffcc00'});
```
```
gauge.refresh(115, null, {valueFontColor: '#00ccff'});
```

This is using the current GitHub version of `justgage.js` (commit: `ce021d6`). Stepping in with a debugger the config settings are updated on line 870, but there appears no code to render these changes after it?

**Comment by toorshia** [comment]
You are correct, there is no code to render changes. This remains to be done.

**Comment by javaday** [comment]
Yea, I just ran into this myself. I am trying to update the min configuration during a refresh. Any ETA on fixing the rendering?

**Comment by toorshia** [comment]
I'd like to roll this out in a month or so.

**Comment by deezone** [comment]
In development...

**Comment by robertsLando** [comment]
Please give a try to JustGage V2 beta now available on NPM 🎉 

https://www.npmjs.com/package/justgage/v/2.0.0-beta.0

`update` function has been enhanced to support more settings
